<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamClass</class>
 <widget class="QMainWindow" name="ExamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="latitude">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="longitude">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="reports_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="messages_list">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="report_description_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_message_line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_report_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_message_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add message</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="report_latitude_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>description</string>
    </property>
   </widget>
   <widget class="QLabel" name="exact_latitude">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>exact_latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="exact_longitude">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>exact_longitude</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="report_longitude_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Exam.qrc"/>
 </resources>
 <connections/>
</ui>
